cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -144,7 +161,9 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
 
     private val archiveMode by mutuallyExclusiveOptions(
         option(
-            help = "Archive the downloaded source code as ZIP files to the output directory. Is ignored if " +
+            help = "Archive the downloaded source code as ZIP files to the output directory. License categories that " +
+                    "are archived can be specified in the ort.conf and be used with the " +
+                    "'--license-classifications-file' flag. By default all packages are archived. Is ignored if " +
```

Comment:
The text could make it more clear that this is about archiving packages with licenses from those categories. Otherwise it can be confusing if you don't know the context, because it's not the license categories which are archived, but package source code.